dartagnan/src/main/java/com/dat3m/dartagnan/verification/model/event/CondJumpModel.java

```diff
@@ -0,0 +1,37 @@
+package com.dat3m.dartagnan.verification.model.event;
+
+import com.dat3m.dartagnan.program.Register.Read;
+import com.dat3m.dartagnan.program.event.Event;
+import com.dat3m.dartagnan.program.event.RegReader;
+import com.dat3m.dartagnan.program.event.core.CondJump;
+import com.dat3m.dartagnan.program.event.core.IfAsJump;
+
+import java.util.Set;
+import java.util.List;
+
+
+public class CondJumpModel extends DefaultEventModel implements RegReaderModel {
+    public CondJumpModel(CondJump event) {
+        super(event);
+    }
+
+    public boolean isGoto() {
+        return ((CondJump) event).isGoto();
+    }
+
+    public boolean isDead() {
+        return ((CondJump) event).isDead();
+    }
```

Comment:
I don't see a use for this.

Reply:
These two are used in `RelationModelManager` to extract `DirectControlDependency`.

Reply:
`.getDependentEvents()` should return an empty list in both cases, no? If so, then checking for goto/dead is unnecessary.